### PR TITLE
Serialization problem when using JsonNodeBinaryType and Hibernate 2nd-level cache

### PR DESCRIPTION
**Pull request by andyflury**
Hi Vlad

Here is the test case that fails. 

Thanks
Andy

### CODE REVIEW

**Comment by andyflury** [comment]
and here is my suggestion on how to fix the issue

File: hibernate-types-43/src/main/java/com/vladmihalcea/hibernate/type/json/internal/JsonNodeTypeDescriptor.java [hunk 1]
```diff
@@ -15,6 +18,17 @@
 
     public JsonNodeTypeDescriptor() {
         super(JsonNode.class, new MutableMutabilityPlan<JsonNode>() {
+
+            @Override
+            public Serializable disassemble(JsonNode value) {
```
Comment (vladmihalcea) [inline hibernate-types-43/src/main/java/com/vladmihalcea/hibernate/type/json/internal/JsonNodeTypeDescriptor.java]: Looks good. The `MutabilityPlan` is the one who takes care of the marshalling/unmarshalling. That's why these two methods are final in the Hibernate Type.

**Comment by vladmihalcea** [comment]
I'm going to integrate it tomorrow.

**Comment by vladmihalcea** [comment]
Applied upstream, thanks

**Comment by andyflury** [comment]
Thank you so much Vlad!

**Comment by vladmihalcea** [comment]
Thank you for the fix. I'll release it after I integrate the other Pull Request for List/Set/Maps.

**Comment by vladmihalcea** [comment]
This feature is now available if you upgrade to 1.2.0.

**Comment by andyflury** [comment]
great! thank you so much Vlad

**Pull request closed without merging**